gcp/modules/backup-exporter/values.yaml

```diff
@@ -0,0 +1,18 @@
+# Default values for backup-exporter
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: google/cloud-sdk
```

Comment:
Please add this image to versions.yml and integrate with all that. See other charts for examples.

Reply:
Moved this task to the description of the next PR. https://github.com/gpii-ops/gpii-infra/pull/349